ssp

```diff
@@ -1754,6 +1755,26 @@ sub print_cpanel_info {
     print_info('cPanel Info: ');
     print_normal($output);
 
+    my %eol = (
+        '56' => { expires => 1509408001, text => 'October 31, 2017' },
+        '58' => { expires => 1501459201, text => 'July 31, 2017' },
+        '60' => { expires => 1509408001, text => 'October 31, 2017' },
+        '62' => { expires => 1522454401, text => 'March 31, 2018' }
+    );
```

Comment:
Seems like this is - going to need to be constantly updated.  At least until we roll out TIERS.json in it's entirety.  Right now we only have 11.62 defined, afaik.
```
      "11.62" : [
         {
            "build" : "11.62.0.21",
            "expires" : 1522454401,
            "is_lts" : 1,
            "is_main" : 1,
            "named" : [
               "stable"
            ]
         }
      ],
```

But ultimately we do want this re-written to where we can just yank down that data, build the data structure on-the-fly, and never have to circle back to re-define all of these versions/expirations.

@cPanelChrisD for now I don't see a way around how this is but it's something to keep in mind for code portability later on.